src/plugins/dataflow/model/dataflow-content.ts

```diff
@@ -294,10 +294,8 @@ export const DataflowContentModel = TileContentModel
       // Time   |  Node 1 | Node 2 | Node 3 etc
       //    0   |   val    | val    |  val
       addAttributeToDataSet(self.dataSet, { name: "Time (sec)" }); //time quantized to nearest sampling rate
-      let insertionOrder = 1;
       self.program.nodes.forEach((n) => { //add attributes based on nodes in tile
-        self.addNewAttrFromNode(n.id, n.name, insertionOrder);
-        insertionOrder ++;
+        n.data.displayNameInsertionOrder && self.addNewAttrFromNode(n.id, n.data.displayNameInsertionOrder);
```

Comment:
I don't understand this code very well, but is there any time here when `n.data.displayNameInsertionOrder` will be falsy? If it is, will that cause problems with the recording?

Reply:
When I take out the left operand, it will complain see attached.  Note that I've changed the variable in the model to `ordedDisplayName`
<img width="445" alt="image" src="https://github.com/concord-consortium/collaborative-learning/assets/521934/ef7cb617-4daa-44f1-bc37-897ee3e60593">